src/Chapter2.hs

```diff
@@ -500,8 +518,12 @@ True
 >>> isThird42 [42, 42, 0, 42]
 False
 -}
-isThird42 = error "isThird42: Not implemented!"
 
+isThird42 :: [Int] -> Bool
+isThird42 [] = False
+isThird42 [x] = False
+isThird42 [x, y] = False
+isThird42 (x:y:z:_) = z == 42
```

Comment:
You can replace `x` and `y` variables with `_` as we don't care about their names. Also, you can pattern match on number 42 directly instead of comparing variable `z`